app/assets/stylesheets/layout.scss

```diff
@@ -64,7 +64,7 @@ body {
 
 .justified-paragraph {
   margin: 30px 0;
-  text-align: justify;
+  text-align: center;
 }
```

Comment:
Don't modify to `text-align: center` directly, you can see the class name `justified-paragraph` , so you shouldn't change css property to `center`.

Insteaed you can create a new class with same details with name `centered-paragraph` and then replace where the class needed